types/cross-domain-utils/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for cross-domain-utils 2.0
+// Project: https://github.com/krakenjs/cross-domain-utils#readme
+// Definitions by: ANovokmet <https://github.com/ANovokmet>
```

Comment:
this is one is fully optional, up to you:
```suggestion
// Definitions by: Ante Novokmet <https://github.com/ANovokmet>
```